metadata.rb

```diff
@@ -0,0 +1,16 @@
+name 'chef-orchestrator'
```

Comment:
maybe just orchestrator ?

Reply:
@tomas-didziokas  - good point, fixed.

Reply:
indeed :+1: `orchestartor` is available https://supermarket.chef.io/cookbooks?utf8=%E2%9C%93&q=orchestrator&platforms%5B%5D=